list2/ex21/index.js

```diff
@@ -0,0 +1,42 @@
+import promptSync from 'prompt-sync';
+const prompt = promptSync();
+
+const WOMEN_WEIGHT_IDEAL_BASE = 62.1;
+const WOMEN_WEIGHT_FORMULA = 44.7;
+const MEN_WEIGHT_IDEAL_BASE = 72.7;
+const MEN_WEIGHT_FORMULA = 58;
+
+
+export function idealWeightFunction() {
+    const gender = {
+        male: 'M',
+        female: 'F'
+    }
```

Comment:
Why are you declaring this inside the function? Isn't it a normal constant?

Reply:
now that I've fixed the function, I've noticed these other problems